husky_gazebo/controller/arm_controller_ur5.yaml

```diff
@@ -0,0 +1,29 @@
+arm_controller:
```

Comment:
I think it uses ROS control. This file is used to tell the ros control plugin what joints will be used. So I think sim & live? ( all robots that use ros_control have this central file now?)